op-batcher/batcher/txmgr.go

```diff
@@ -121,6 +121,8 @@ func (t *TransactionManager) CraftTx(ctx context.Context, data []byte) (*types.T
 	}
 	rawTx.Gas = gas
 
+	ctx, cancel = context.WithTimeout(ctx, networkTimeout)
```

Comment:
I'm re-using `networkTimeout`, but that will be a little at-odds for non-networked signing setups. Open to suggestions if there's something better.